.github/workflows/widget-cd.yml

```diff
@@ -30,6 +30,15 @@ jobs:
     runs-on: [ubuntu-latest]
     needs: [build-widget]
     steps:
+      - name: Determine target tenant
+        id: determine-target-tenant
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            echo "tenantUrl=${{ github.event.inputs.tenant_url }}" >> $GITHUB_OUTPUT
+          elif [[ "${{ github.event_name }}" != "workflow_dispatch" ]]; then
+            echo "tenantUrl=${{ secrets.TENANT_URL }}" >> $GITHUB_OUTPUT
```

Comment:
Why tenant URL is added under secrets?

Reply:
It is a public repo, I cannot expose our tenant details in workflow

Reply:
Got it.